docs/reference/developer/release-process.md

```diff
@@ -66,62 +70,29 @@ yarn upgrade
 
 > **Note**: this image is not being built automatically. Consequently, a [manual build process](./end-to-end-tests.md#building-the-kubeappsintegration-tests-image) _must_ be triggered if you happen to upgrade the integration image or its dependencies.
 
-### 0.3 - Development chart
-
-Even though the official [Bitnami chart](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps) is automatically able to retrieve the latest dependency versions, we still need to sync the versions declared in our own development chart.
-
-#### 0.3.1 - Chart images
-
-Currently, the [values.yaml](../../chart/kubeapps/values.yaml) uses the following container images:
-
-- [bitnami/nginx](https://hub.docker.com/r/bitnami/nginx/tags)
-- [bitnami/oauth2-proxy](https://hub.docker.com/r/bitnami/oauth2-proxy/tags)
-
-> As part of this release process, these image tags _must_ be updated to the latest minor version. In case of a major version, the change _should_ be tracked in a separate PR.
-
-#### 0.3.2 - Chart dependencies
-
-The chart [dependencies](../../chart/kubeapps/Chart.yaml) _must_ be checked to ensure the version includes the latest dependent charts.
-
-- Check if the latest versions are already included by running:
-
-```bash
-helm dep list ./chart/kubeapps
-```
-
-- If they are not, run this other command to update the `requirements.lock` file:
-
-```bash
-# It requires Helm >=3.7.0, otherwise, just run `helm dep update ./chart/kubeapps`
-helm repo update bitnami
-helm dep update --skip-refresh ./chart/kubeapps
-```
-
-> As part of this release process, the chart dependencies _must_ be updated to the latest versions. In case of a major version, the change _should_ be tracked in a separate PR.
-
-### 0.4 - Protobuf dependencies and autogenerated code
```

Comment:
I've removed this section because we get this automatically from the merges of the bitnami chart now, don't we?

Reply:
I think so. In the last 2-3 days we have received from Bitnami almost 1 chart update per day.